From 4ac82cdf92ae6c0e2ff9c1238bfe28cfcefea99c Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Fri, 3 Jan 2014 10:04:39 +0100
Subject: [PATCH] acc: added note about MySQL InnoDB not supporting INSERT
 DELAYED

- reported by Daniel Tryba

(cherry picked from commit d26eff22c8a543c63a40cd7d91bb3c70f04aabc6)
---
 modules/acc/doc/acc_admin.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/modules/acc/doc/acc_admin.xml b/modules/acc/doc/acc_admin.xml
index 97858be..33ca1f8 100644
--- a/modules/acc/doc/acc_admin.xml
+++ b/modules/acc/doc/acc_admin.xml
@@ -1093,7 +1093,9 @@ modparam("acc", "db_extra", "ct=$hdr(Content-type); email=$avp(s:email)")
 		<para>
 		If set to 1, use INSERT DELAYED to add records to accounting tables
 		when the DB driver has support for it. If no INSERT DELAYED support
-		is offered by DB driver, then standard INSERT is used.
+		is offered by DB driver, then standard INSERT is used. Beware that
+		MySQL InnoDB engine doesn't support INSERT DELAYED, thus be sure
+		the acc tables are defined with different type (e.g., MyISAM).
 		</para>
 		<para>
 		Default value is 0 (no INSERT DELAYED).
-- 
1.8.5.2

